stories/card/card.stories.js

```diff
@@ -338,27 +342,27 @@ export const listCard = () => `
             <div class="fd-card__content" role="group" aria-label="Card Content">
                 <ul class="fd-list fd-list--no-border" role="list">
                     <li role="listitem" tabindex="0" class="fd-list__item">
-                        <span class="fd-list__icon sap-icon--cart"></span>
+                        <span aria-hidden="true" aria-hidden="true" class="fd-list__icon sap-icon--cart"></span>
                         <span class="fd-list__title">List item 1</span>
```

Comment:
Why is aria-hidden added here? Shouldn't we put the icon in a separate <i> tag for consistency?